erpnext_documentation/www/docs/user/manual/en/human-resources/leave-policy-assignment.md

```diff
@@ -0,0 +1,74 @@
+# Leave Policy Assignment
+
+> Introduced in Version 13
+
+Leave Policy Assignment in ERPNext is used to grant leaves to employees. To access Leave policy assignment, go to:
+
+> Home > Human Resources > Leaves > Leave Policy Assignment
+
+## 1. Prerequisites
+
+Before creating a Leave Policy Assignment, it is advisable to create the following:
+
+* [Employee](/docs/user/manual/en/human-resources/employee)
+* [Leave Policy](/docs/user/manual/en/human-resources/leave-policy)
+
+## 2. How to create a Leave Policy Assignment
+
+1. Go to Leave Policy Assignment, click on New.
+1. Select Employee and Leave Policy.
+1. Select Assignment based on if required.
+    * If an assignment is based on Leave period, You need to select Leave period. The Effective From and Effective To will be set automatically.
+    * If an assignment is based on Joining Date, The Effective From date will be set to Employee's Date of Joining.
+    * If an assignment is based on is blank, You need to set Effective From and Effective To Manually.
+1. Save and Submit.
+
+<img class="screenshot" alt="Leave Policy Assignment"
+	src="{{docs_base_url}}/assets/img/human-resources/leave-policy-assignment.png">
+
+## 3. Granting Leave
+
+Once the information is saved, the Leave Policy Assignment will also be used as a tool to help you grant leaves for employees. The **Grant Leave** button will appear at the right top corner.
+
+
+<img class="screenshot" alt="Grant Leaves"
+	src="{{docs_base_url}}/assets/img/human-resources/leave-policy-assignment-grant-leave.png">
+
+On click, The Leave allocation will be created automatically based on [Leave Policy](/docs/user/manual/en/human-resources/leave-policy) as shown below.
+
+<img class="screenshot" alt="Leave Allocations"
+	src="{{docs_base_url}}/assets/img/human-resources/granted-leaves.png">
+
+>**Note:** The Grant Leave button will appear only then if, No leaves are granted or there is no allocation against current Leave Policy Assignment.
+
+## 4. Automatic Allocate Leaves Based On Leave Policy
+
+To enable Automatic allocation for your Leave Policy Assignment you need to Enable [Automatic Allocate Leaves Based On Leave Policy ](/docs/user/manual/en/human-resources/hr-settings#37-automatic-allocate-leaves-based-on-leave-policy) checkbox in HR Settings. Our Scheduler Run background Job and check the Effective From Date in Leave Policy Assignment and create Leave allocation automatically.
```

Comment:
```suggestion
To enable automatic allocation of leaves based on the Leave Policy Assignment, enable [Automatic Allocate Leaves Based On Leave Policy ](/docs/user/manual/en/human-resources/hr-settings#37-automatic-allocate-leaves-based-on-leave-policy) checkbox in HR Settings. The scheduler then runs a background job and checks the Effective From date in the Leave Policy Assignment to create leave allocations automatically.
```

Reply:
@Anurag810 change the checkbox name to "Automatically Allocate Leaves Based On Leave Policy"